content/references/data-api.md

```diff
@@ -2331,6 +2456,107 @@ Response:
 
 
 
+
+## Get Fee Stats
```

Comment:
How are these methods organized? Probably outside the scope of this PR but we might want to readdress this. I was expecting them to be listed in alpha order but got...something else.

Reply:
They're partly organized by the URL of the method, but not completely.

I agree we should fix it in a separate task.